src/actions/ScanActions.js

```diff
@@ -159,8 +158,7 @@ export const parseScannedUri = (data: string, customErrorTitle?: string, customE
 
       if (isPaymentProtocolUri(parsedUri)) {
         // BIP70 URI
-        // $FlowFixMe
-        return dispatch(paymentProtocolUriReceived(parsedUri))
+        // BUSTED!
```

Comment:
We cannot leave these two in a busted state. We need to somehow route these two URI's to the new Send scene. This is the reason we haven't already removed the SendConfirmation scene - doing so would break deep linking.